Automattic/wp-calypso
# i18n: Translating SSR themes section

#### Changes proposed in this Pull Request

* Add themes section paths with `lang` parameter.
* Setup locale for logged-out themes section.
* Localize internal themes links when logged-out.

**Preview:**
![image](https://user-images.githubusercontent.com/2722412/118491604-d4fead80-b727-11eb-9eea-fcb23a3c4f6e.png)

![image](https://user-images.githubusercontent.com/2722412/118491613-d8923480-b727-11eb-9640-623ed5eefd30.png)


#### Testing instructions

* Use `calypso.live` build or boot locally with `BUILD_TRANSLATION_CHUNKS=true ENABLE_FEATURES=use-translation-chunks yarn start`
* Test logged-out `/{:locale}/themes`, `/{:locale}/theme/{:themeSlug}` and their respective sub screens and confirm server rendering works as expected.
* Confirm the changes don't affect logged-in themes section. Note that the layout is bugged in development when logged in - pd2qbF-P-p2#comment-20.
* Review code changes
